src/nytt/Nytt.less

```diff
@@ -91,7 +91,7 @@
         margin-bottom: 0.125rem;
     }
 
-    &__artikkelinnhold {
+    &__artikkelinnhold.typo-normal {
```

Comment:
Rekbis-stilling importerer stil til .typo-normal tusen ganger, så må spesifisere ytterligere for at regelen ikke skal overskrives der